homeassistant/components/flipr/config_flow.py

```diff
@@ -0,0 +1,95 @@
+"""Config flow for Flipr integration."""
+import logging
+from typing import List
+
+from flipr_api import FliprAPIRestClient
+from requests.exceptions import HTTPError, Timeout
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+
+from .const import CONF_FLIPR_IDS
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Flipr."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    def __init__(self):
+        """Initialize Flipr config flow."""
+        self._username = None
+        self._password = None
+        self._flipr_ids = None
```

Comment:
```suggestion
    _username: str | None = None
    _password: str | None = None
    _flipr_ids: str | None = None
```

Reply:
This does not work as is since python 3.10 is not the current version (at least in the devcontainer of the upstream dev branch). So I used the old style with typing.Union for now.